src/server.c

```diff
@@ -48,7 +49,7 @@ int dqlite__init(struct dqlite_node *d,
 	}
 	rv = raft_uv_init(&d->raft_io, &d->loop, dir, &d->raft_transport);
 	if (rv != 0) {
-		/* TODO: better error reporting */
+		snprintf(d->errmsg, DQLITE_ERRMSG_BUF_SIZE, "raft_uv_init(): failed");
```

Comment:
Similarly, we could provide more details using something like:

```C
snprintf(d->errmsg, DQLITE_ERRMSG_BUF_SIZE, "raft_uv_init(): %s", d->raft_io.errmsg);
```

See the implementation of `raft_uv_init()` for details.